tests/darwin/exporter/formats/export_darwin_test.py

```diff
@@ -128,3 +129,8 @@ def test_complete_annotation_file_with_bounding_box_and_tag_v2():
     }
 
     assert build_image_annotation(annotation_file, "Test team") == expected_output
+
+
+def test_parse_annotations_with_missing_required_properties():
+    path = Path("tests/darwin/data/annotation_with_missing_required_properties.json")
+    annotaions = parse_darwin_json(path=path)
```

Comment:
This tests that `SelectedProperty` objects can be created even if `frame_index` & `value` fields are `None`